source/times.js

```diff
@@ -26,17 +25,18 @@ import _curry2 from './internal/_curry2.js';
  */
 var times = _curry2(function times(fn, n) {
   var len = Number(n);
-  var idx = 0;
   var list;
 
   if (len < 0 || isNaN(len)) {
     throw new RangeError('n must be a non-negative number');
   }
-  list = new Array(len);
-  while (idx < len) {
-    list[idx] = fn(idx);
-    idx += 1;
+
+  list = [];
+  for (let x = 0; x < len; x += 1) {
+    list[x] = fn(x);
```

Comment:
I _think_ `.push` would be faster (on paper) and would also avoid potential weird case if the user supplied a float as `n`.

Reply:
umm, why `push` would be faster?

Reply:
I don't know why it is faster just that it seems to be the case (or used to be) in some browsers. Most likely irrelevant these days though but as it allows you to avoid managing the indexes (and avoid sparse arrays) _and_ avoid issues with non-integer indexes I see little reason not to use it.

Reply:
++